dev/scripts/src/alluxio.org/build-distribution/cmd/generate-tarball.go

```diff
@@ -305,11 +305,16 @@ func generateTarball(hadoopClients []string) error {
 	fmt.Printf("Creating %s:\n", tarball)
 
 	for _, dir := range []string{
-		"assembly", "client", "logs", "integration/fuse", "integration/checker",
+		"assembly", "client", "logs", "integration/fuse", "integration/checker", "logs/user",
 	} {
 		mkdir(filepath.Join(dstPath, dir))
 	}
 
+    err := os.Chmod(filepath.Join(dstPath, "logs/user"), 0777)
```

Comment:
the Gopher idiom for this common pattern is
```go
if err := os.Chmod(filepath.Join(dstPath, "logs/user"), 0777); err != nil {
    return err
}

Reply:
Noted and fixed.